pydax/loaders/_table.py

```diff
@@ -55,9 +57,18 @@ def load(self, path: Union[_typing.PathLike, Dict[str, str]], options: SchemaDic
             else:
                 dtypes[column] = type_
 
+        names = None
+        header = None
+        if options.get('header', True):
```

Comment:
Based on the document, do you mean

```suggestion
        if options.get('header', True) is not False:
```

Or you can actually make the function based on Python's evaluation of whether the value is true or false rather than deciding whether it is exactly `False` or not.

Reply:
I think it might be confusing if we make the function based on Python's evaluation. If we accidentally set `header` to `''`. Maybe we could rename the key to `no_header` and then use Python's evaluation?

Reply:
Sounds good.